package.json

```diff
@@ -1,46 +1,59 @@
 {
   "name": "@exam/root",
   "type": "module",
+  "packageManager": "pnpm@8.15.9+sha256.daa27a0b541bc635323ff96c2ded995467ff9fe6d69ff67021558aa9ad9dcc36",
```

Comment:
Should we use pnpm v9?

Reply:
Fixed [2e8ddae build: corepack use pnpm@latest](https://github.com/chatbotgang/exam-local-chat/pull/10/commits/2e8ddae9ddf5d63d5e6d373f2c214604786a6268)